spec/tilebag_spec.rb

```diff
@@ -0,0 +1,39 @@
+require_relative 'spec_helper'
+
+describe "TileBag" do
+  describe "#initialize" do
+    it "Should create instance of the Tilebag" do
+      tilebag = Scrabble::TileBag.new
+      tilebag.must_be_kind_of Scrabble::TileBag
+    end
+
+    it "Should create a collection of default tiles" do
+      tilebag = Scrabble::TileBag.new
+      tilebag.tiles_remaining.must_equal 98
+    end
+  end
+
+  describe "#draw_tiles method" do
+    it "Should make sure a max of 7 tiles drawn" do
+  	tilebag = Scrabble::TileBag.new
```

Comment:
You're missing a couple of interesting edge cases here:
- What happens if you attempt to draw 0 tiles?
- Are you able to draw the bag all the way down to 0 tiles remaining?
- What happens when you attempt to draw more tiles than are in the bag?